src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -736,6 +737,122 @@ def get_inference_statistics(self,
         except grpc.RpcError as rpc_error:
             raise_error_grpc(rpc_error)
 
+    def update_trace_setting(self,
+                             model_name="",
+                             setting={},
+                             headers=None,
+                             as_json=False):
+        """Update the trace setting for the specified model name, or global trace
+        setting if model name is not given. Returns the trace setting after
+        the update.
+
+        Parameters
+        ----------
+        model_name : str
+            The name of the model to get trace setting. The default value is
+            an empty string, which means global trace setting will
+            be returned.
+        setting: dict
+            The new trace setting values. Only the setting listed will be
+            updated, and the value of None will clear the previously specified
```

Comment:
".. updated. If a trace setting is listed in the dictionary with a value of 'None', that setting will be cleared."